docs/README.md

```diff
@@ -18,60 +25,75 @@ This benchmark requires the followings:
-
-| name                  | description | default |
-|:----------------------|:------------|:--------|
-| `--num-threads`       | Number of threads for benchmarking. | 1 |
-| `--num-warehouses`    | Number of warehouses (scale factor) for benchmarking. | 1 |
-| `--duration`          | Duration of benchmark (in seconds). | 200 |
-| `--ramp-up-time`      | Duration of ramp up time before benchmark (in seconds). | 30 |
-| `--use-table-index`   | Whether or not to use a generic table-based secondary index instead of ScalarDB's secondary index. Data must be loaded using the same option. | false |
-| `--rate-new-order`    | Percentage of new-order transaction.     | 45 |
-| `--rate-payment`      | Percentage of payment transaction.       | 43 |
-| `--rate-order-status` | Percentage of order-status transaction.  | 4  |
-| `--rate-delivery`     | Percentage of delivery transaction.      | 4  |
-| `--rate-stock-level`  | Percentage of stock-level transaction.   | 4  |
-| `--np-only`           | Run benchmark with only new-order and payment transactions (50% each). | false |
+### Load and run
+
+1. Prepare a configuration file
+    - A configuration file requires at least the locations of workload modules to run and the database configuration. The following example shows the case for running TPC-C benchmark. The database configuration should be matched with the benchmark environment set up above. You can use the ScalarDB property file instead of specifying each configuration item. If the `config_file` is specified, all other configuration items will be ignored.
+      ```
+      [modules]
+      [modules.preprocessor]
+      name = "com.scalar.db.benchmarks.tpcc.TpccLoader"
+      path = "./build/libs/scalardb-benchmarks-all.jar"
+      [modules.processor]
+      name = "com.scalar.db.benchmarks.tpcc.TpccBench"
+      path = "./build/libs/scalardb-benchmarks-all.jar"
+      [modules.postprocessor]
+      name = "com.scalar.db.benchmarks.tpcc.TpccReporter"
+      path = "./build/libs/scalardb-benchmarks-all.jar"
+ 
+      [database_config]
+      contact_points = "localhost"
+      contact_port = 9042
+      username = "cassandra"
+      password = "cassandra"
+      storage = "cassandra"
+      #config_file = "/path/to/scalardb.properties"
+      ```
+    - You can define static parameters to pass to modules in the file. For details, see example configuration files such as `tpcc-benchmark-config.toml` and available parameters in [the following section](#common-parameters).
+2. Run a benchmark
+   ```
+   ${kelpie}/bin/kelpie --config your_config.toml
+   ```
+    - `${kelpie}` is a Kelpie directory, which is extracted from the archive you downloaded [above](#prerequisites).
+    - There are other options such as `--only-pre` (i.e., registering certificates and contracts and loading data) and `--only-process` (i.e., running benchmark), which run only the specified process. `--except-pre` and `--except-process` run a job without the specified process.
```

Comment:
The descriptions of the options seem to be for ScalarDL.

Reply:
Good catch, thank you! Fixed in 18e61c9.